src/platform/plugins/shared/data_view_field_editor/public/components/preview/field_list/field_list.scss

```diff
@@ -20,12 +20,6 @@ $previewShowMoreHeight: 40px; /* [2] */
     align-items: center;
     overflow: hidden;
 
-    &--highlighted {
```

Comment:
Should we get rid of the `indexPatternFieldEditor__previewFieldList__item--highlighted` class from the component if it's no longer used for styling?

Reply:
Unfortunately the same issue with tests - until we migrate to test subjects it fails without this class. I'll create a ticket for that, unless we want to do it within scope if this one?

Reply:
It's the same case as in the other component - tests fail without this class. I'm happy to migrate it to test subj in a separate ticket 😁  does it sound good?

Reply:
Ah ok, I did a quick search in the codebase and didn't see other references so assumed it'd be fine, but handling it separately is fine by me too.

Reply:
Resolving this one then and will handle data test subj in a different PR 😁